src/CMakeLists.txt

```diff
@@ -12,21 +12,15 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-add_library(VulkanLayerUtils STATIC)
-add_library(Vulkan::LayerUtils ALIAS VulkanLayerUtils)
-
 target_sources(VulkanLayerUtils PRIVATE
+   CMakeLists.txt
    vk_layer_settings.cpp
-   vk_layer_settings.h
 )
-target_link_Libraries(VulkanLayerUtils PRIVATE Vulkan::Headers)
 
-if(WIN32)
-   target_compile_definitions(VulkanLayerUtils PUBLIC _CRT_SECURE_NO_WARNINGS)
-endif()
+# NOTE: Because Vulkan::Headers header files are exposed in the public facing interface
+# we must expose this library as public to users.
+target_link_Libraries(VulkanLayerUtils PUBLIC Vulkan::Headers)
 
-if(VUL_TESTS)
-    enable_testing()
-    add_subdirectory(tests)
+if(WIN32)
+   target_compile_definitions(VulkanLayerUtils PRIVATE _CRT_SECURE_NO_WARNINGS)
```

Comment:
This should be PRIVATE since clients of this library shouldn't have to disable CRT secure warnings.

Reply:
Here are the properties currently being exposed in the interface of the library:

```
# Create imported target Vulkan::LayerUtils
add_library(Vulkan::LayerUtils STATIC IMPORTED)

set_target_properties(Vulkan::LayerUtils PROPERTIES
  INTERFACE_COMPILE_FEATURES "cxx_std_17"
  INTERFACE_INCLUDE_DIRECTORIES "${_IMPORT_PREFIX}/include"
  INTERFACE_LINK_LIBRARIES "Vulkan::Headers"
)
```